watertap/unit_models/tests/test_electrodialysis_0d.py

```diff
@@ -0,0 +1,672 @@
+###############################################################################
+# ProteusLib Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/nawi-hub/proteuslib/"
+#
+###############################################################################
```

Comment:
This is the old copyright statement from ProteusLib. This needs to be replaced with the copyright from WaterTAP (the one you used in the unit model). If this was copy-pasted from another file, then it needs to be changed in that other file as well. 

If you could, also replace the copyright statement in the file you got this from in this PR.

Reply:
Yes, I also corrected the file "test_coag_..."  where I took it from .